solver/llbsolver/history.go

```diff
@@ -433,11 +433,11 @@ func (h *HistoryQueue) Status(ctx context.Context, ref string, st chan<- *client
 	if err := h.opt.DB.View(func(tx *bolt.Tx) error {
 		b := tx.Bucket([]byte(recordsBucket))
 		if b == nil {
-			return os.ErrNotExist
+			return errors.Wrapf(os.ErrNotExist, "failed to retreive bucket %s", recordsBucket)
```

Comment:
Ah didn't catch this typo :sweat_smile:

```suggestion
			return errors.Wrapf(os.ErrNotExist, "failed to retrieve bucket %s", recordsBucket)
```